components/suggest/src/db.rs

```diff
@@ -140,17 +140,17 @@ impl<'a> SuggestDao<'a> {
         {
             (self.conn.prepare_cached(
                 &format!(
-                    "SELECT s.id, k.rank, s.title, s.url, s.provider, NULL as confidence, NULL as keyword_suffix
+                    "SELECT s.id, k.rank, s.title, s.url, s.provider, s.score, NULL as confidence, NULL as keyword_suffix
                      FROM suggestions s
                      JOIN keywords k ON k.suggestion_id = s.id
                      WHERE s.provider IN ({}) AND
                            k.keyword = :keyword
                      UNION ALL
-                     SELECT s.id, k.rank, s.title, s.url, s.provider, k.confidence, k.keyword_suffix
+                     SELECT s.id, k.rank, s.title, s.url, s.provider, s.score, k.confidence, k.keyword_suffix
                      FROM suggestions s
                      JOIN prefix_keywords k ON k.suggestion_id = s.id
                      WHERE k.keyword_prefix = :keyword_prefix
-                     ORDER BY s.provider
+                     ORDER BY s.score DESC
```

Comment:
I think you’ll want `IFNULL(s.score, 0.2)` here, too, or any suggestions without a score will sort last.

Could you please make `0.2` a constant, too? It’ll help clarify that it’s the default score, so that we aren’t embedding magic numbers in the query. Thanks!

Reply:
Another tip: If you write `IFNULL(s.score, 0.2) AS score` in the `SELECT`, you can just write `ORDER BY score DESC` here! 🎉 `ORDER BY` can refer to any columns or aliased expressions from the `SELECT`.

SQLite's [documentation for SELECT](https://sqlite.org/lang_select.html) is a long read, but I promise, it's _super_ worth it for helping the syntax feel less arcane.